galasa-ui/src/actions/getTestRuns.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+'use server';
+
+import { ResultArchiveStoreAPIApi, Run, RunResults } from "@/generated/galasaapi";
+import { createAuthenticatedApiConfiguration } from "@/utils/api";
+import * as Constants from "@/utils/constants";
+
+/**
+ * Fetches test runs from the Result Archive Store (RAS) for the last 24 hours.
+ * 
+ * @param {string} loginId - The login ID of the user.
+ * @returns {Promise<Run[]>} - A promise that resolves to an array of Run objects.
+ */
+export const fetchMyTestRunsForLastDay  = async (): Promise<Run[]> => {
```

Comment:
The `loginId` parameter can be removed from the comment since the function takes no parameters :)

Also, the function name says `MyTestRuns` but can now be something like `fetchAllTestRunsForLastDay`